wiki/People/Beatmap_Nominators/en.md

```diff
@@ -29,6 +29,7 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
 ### What Qualifies as a Mod Post
 + Upon signing up you will be asked to provide the total amount of Kudosu as well as the amount of mod posts within the last 3 months, sorted by month upon signing up. So if the announcement is posted in April the mod posts for January, February and March will be relevant. The relevant months will also be listed on the submission form.
```

Comment:
Add commas after April and February
regarding feb's comma, it's an oxford comma which should be used.